src/internal_rep.rs

```diff
@@ -1387,6 +1422,27 @@ impl<'a> FunctionInfo<'a> {
         self.body = Some(new_body);
         errors.into_result(())
     }
+
+    // Generate the sexp for a synthetic alias function calling the real function
+    pub fn generate_synthetic_alias_call(&self, alias_cil_name: &str) -> sexp::Sexp {
```

Comment:
Do we really want a `call baz-read` or a *copy* of baz-read, something like the synthetic resources (associations), which would be generic?

Reply:
I'm not sure I see the benefit in copying `baz-read` here.  Calling it reduces the size of the outputted policy.  The original function is guaranteed to exist because there's no way to create an alias without the "real" function.  Is there some benefit of making a new copy rather than just calling the original that I'm missing?